src/screenComponents/mineSweeper.h

```diff
@@ -0,0 +1,35 @@
+/** An implementation of mineSweeper for use as a hacking minigame.
+ *  Original implementation by https://github.com/daid
+ */
+
+#ifndef MINESWEEPER_H
+#define MINESWEEPER_H
+
+#include "miniGame.h"
+#include "gui/gui2_togglebutton.h"
+#include <vector>
+#include <memory>
+
+
+class MineSweeper : public MiniGame {
+  public:
+    MineSweeper(GuiPanel* owner, GuiHackingDialog* parent, int difficulty);
+    virtual void reset();
```

Comment:
As these virtual functions override the base class, it would be good to mark them as "override":
https://en.cppreference.com/w/cpp/language/override

Reply:
Agreed, done.